scripts/makeupdates

```diff
@@ -27,6 +27,9 @@ import six
 import subprocess
 import sys
 
+# The Python site-packages path for blivet.
+PYTHON_LIB_PATH = "./usr/lib/python3.9/site-packages/"
```

Comment:
I am not sure how to handle different versions of Python in your case. Anaconda will change the version per branch.